ProfitSite/src/main/java/bsu/fpmi/profit/LoginServlet.java

```diff
@@ -0,0 +1,57 @@
+package bsu.fpmi.profit;
+
+import com.google.gson.Gson;
+import org.apache.commons.codec.binary.Base64;
+
+import javax.servlet.*;
+import javax.servlet.http.*;
+import java.io.IOException;
+
+public class LoginServlet extends HttpServlet {
+    private final String URL = "jdbc:mysql://localhost:3306/profit";
+    private final String USER = "root";
+    private final String PASSWORD_DB = "123456789";
+    private static final String USERNAME = "username";
```

Comment:
Please do not store credentials in a servlet, no one will ever know where to look for them) Use a dedicated resource file, or at least make them constants in frames of the DB subsystem